lib/beaker-benchmark/helpers.rb

```diff
@@ -42,61 +46,67 @@ def measure_perf_on(infrastructure_host, action_name, include_processes=false, p
           # Append action name to test case name if test name is available
           action_name = metadata[:case][:name] + "_#{action_name}" if defined? metadata && metadata[:case] && metadata[:case][:name]
 
-          atop_log = 'atop_log.log'
-
-          start_monitoring(infrastructure_host, atop_log, include_processes)
+          start_monitoring(infrastructure_host, action_name, include_processes)
 
           yield
 
-          stop_monitoring(infrastructure_host, action_name, atop_log, include_processes.nil? ? nil : process_regex)
+          stop_monitoring(infrastructure_host, include_processes.nil? ? nil : process_regex)
         end
 
         def setup_atop(infrastructure_host)
-          @benchmark_tmpdir = Dir.mktmpdir
           # Only install atop once per host
           unless infrastructure_host.check_for_package('atop')
-            add_el_extras(infrastructure_host)
+            add_el_extras(infrastructure_host, @options)
```

Comment:
where is this @options coming from?  I don't see it anywhere else in this file.

Reply:
That is the options that are available from Beaker::TestCase.  Basically I had to change this because if you don't install PE (which I'm not in my acceptance test) then it couldn't find add_el_extras.  There is a base add_el_extras in include Beaker::HostPrebuiltSteps which is what beaker-pe calls into, but it takes this required options parameter.